app/app.go

```diff
@@ -59,8 +59,8 @@ import (
 )
 
 const (
-	AccountAddressPrefix = "cosmos"
-	Name                 = "poktroll"
+	AccountAddressPrefix = "pokt"
+	Name                 = "pocket"
```

Comment:
Is this going to change the binary to `pocketd` instead of `poktrolld`. 

It'll impact the documentation if so.

Reply:
I didn't change anything - these values are what we have in main branch: https://github.com/pokt-network/poktroll/blob/main/app/app.go#L141

Reply:
The main branch has `pocket` but this is a PR against `migration/base`, which has `poktroll`.

Is this intended?

cc @bryanchriswhite

Reply:
I think we actually do want to keep `Name`  as `poktroll`. I'm noticing that it doesn't seem to have an effect on the binary path `.../poktrolld`, however it does have an impact on the `--help` text as well as the default home path and environment variable prefixes from viper:
![image](https://github.com/pokt-network/poktroll/assets/600733/6e2492c3-f7e0-4895-aefd-abdbdd92db2e)